exporter/datadogexporter/internal/metrics/runtime_metrics_mappings.go

```diff
@@ -0,0 +1,17 @@
+package metrics
+
+// goRuntimeMetricsMappings defines the mappings from OTel runtime metric names to their
+// equivalent Datadog runtime metric names
+var goRuntimeMetricsMappings = map[string]string{
```

Comment:
Why not call this `runtimeMetricsMappings` instead and extend the list later on with all languages? Not just Go. I may be missing context here, so do fill me in if that's the case.